CONTRIBUTING.md

```diff
@@ -82,34 +86,34 @@ Sometimes small mistakes fall through through the cracks. So if you see an error
 
 #### New articles or pages
 
-Have an idea for a new page or article, or even a series of articles? The first step is to create an issue with your suggestions. The article you are suggesting may have been discussed before, or somebody else could already be working on it.
+Have an idea for a new page or article, or even a series of articles? The first step is to [create an issue](https://github.com/ipfs/docs/issues/new/choose) with your suggestions. The article you are suggesting may have been discussed before, or somebody else could already be working on it.
 
 Once an issue has been created, it can be discussed by the community and IPFS project leads. Once approved, you are free to start writing and implementing your documentation into the repository. Internal IPFS project leads from Protocol Labs review all pull-requests.
 
 ### File an issue
 
 Follow these steps to learn how to file an issue with the IPFS project.
 
-1. Go to the [Issues](https://github.com/ipfs/docs/issues/) tab in the [`ipfs/docs`](https://github.com/ipfs/docs) repository.
+1. Go to the [Issues](https://github.com/ipfs/docs/issues/) tab in the [`ipfs/docs`](https://github.com/ipfs/docs) repository. (Note: docs *issues* are tracked in `ipfs/docs`, but the *code and content* that power the docs beta site are in the `ipfs/ipfs-docs-v2` repo until the legacy docs site is deprecated -- currently slated for mid-2020.)
```

Comment:
Could we take the _Note: docs *issues* are tracked_ section out of the brackets `()`? Auto-translators and screen readers can have a hard time with them, especially with longer sentences like this one.

Reply:
Done.